Tests/UnitTests/Networking/Responses/OfferingsDecodingTests.swift

```diff
@@ -48,10 +49,48 @@ class OfferingsDecodingTests: BaseHTTPResponseTest {
     }
 
     func testDecodesSecondOffering() throws {
-        let offering = try XCTUnwrap(self.response.offerings.last)
+        let offering = try XCTUnwrap(self.response.offerings[1])
```

Comment:
This will make the test crash.  I'll help you on Monday, we'll need to define a new method that returns an Optional for an index.